pages/socios/crear.tsx

```diff
@@ -22,7 +22,16 @@ const Create: NextPage = () => {
         position: 'Socio',
         type: 'MEMBER',
       }
-      await axios.post('/api/members', member)
+      const user = await axios.post('/api/members', member)
+      const { data } = user.data
+      const { startDate, memberId } = data
```

Comment:
Add guards here to avoid problems when data is not sent

Reply:
Thanks for your review.
Is it ideal to throw an error in this way?

Reply:
Yes, as an engineer you have to be aware that errors can happen, and prepare for them